### PR TITLE
docs: fix vertical bar in tables

### PR DESCRIPTION
**Pull request by fz6m**

### CODE REVIEW

File: docs/.vitepress/theme/index.js [hunk 1]
```diff
@@ -1,5 +1,6 @@
 import DefaultTheme from 'vitepress/dist/client/theme-default'
 import Layout from './Layout.vue'
+import './styles/index.css'
```
Comment (posva) [inline docs/.vitepress/theme/index.js:3]: I pushed a change in CSS for this, you can remove it. It will avoid adding `div`s

**Review by posva — CHANGES_REQUESTED** [review]
Thank you! Can you rebase from master? I added the needed CSS to avoid having to use `div`s

**Review by posva — APPROVED** [review]
Thanks!

File: docs/.vitepress/theme/index.js (hunk 1 above)
Comment (fz6m) [reply docs/.vitepress/theme/index.js]: Great. I deleted this css file

**Pull request merged**